gruntfile.js

```diff
@@ -62,73 +62,73 @@ module.exports = function(grunt) {
     },
     clean: ['dist'],
     connect: {
```

Comment:
Gruntfile was changed. Confirm if it's correct, since it was commited by mistake.